test/types/coerce/allNumericsBinaryRealImagComplex-errors.good

```diff
@@ -1,9 +1,8 @@
-allNumericsBinaryRealImagComplex-errors.chpl:28: In function 'callF':
-allNumericsBinaryRealImagComplex-errors.chpl:32: error: ambiguous call 'f(int(8), int(64))'
+allNumericsBinaryRealImagComplex-errors.chpl:23: In function 'callF':
+allNumericsBinaryRealImagComplex-errors.chpl:27: error: ambiguous call 'f(int(8), int(64))'
```

Comment:
Huh... I'm surprised that this doesn't prefer a `real, real` overload.  Is the reason because the 8-bit arg prefers the `real(32)` and the 64-bit arg prefers `real(64)`?

Reply:
<details>

```
allNumericsBinaryRealImagComplex-errors.chpl:27: note: call: f(int(8), int(64))
```

```
allNumericsBinaryRealImagComplex-errors.chpl:1: note: candidates are: f(x: real(32), y: real(32)) [182416]
allNumericsBinaryRealImagComplex-errors.chpl:4: note:                 f(x: real(64), y: real(64)) [182450]
allNumericsBinaryRealImagComplex-errors.chpl:15: note:                 f(x: complex(64), y: complex(64)) [182546]
allNumericsBinaryRealImagComplex-errors.chpl:18: note:                 f(x: complex(128), y: complex(128)) [182578]
```

```
##########################
# Considering function 0 #
##########################

f(x: real(32), y: real(32)) [182416]

Comparing to function 1
-----------------------
f(x: real(64), y: real(64)) [182450]

Looking at argument 0
Actual's type: int(8)
Formal 1's type: real(32)
Actual requires coercion to match formal 1
Formal 2's type: real(64)
Actual requires coercion to match formal 2
can dispatch: Fn 0 is non-param preferred

Looking at argument 1
Actual's type: int(64)
Formal 1's type: real(32)
Actual requires coercion to match formal 1
Formal 2's type: real(64)
Actual requires coercion to match formal 2
same numeric width: Fn 1 is non-param preferred

W: Fn 0 and Fn 1 are incomparable
X: Fn 0 is incomparable with Fn 1
```

</details>

Yes, the `int(8)` arg prefers `real(32)` due to the "can dispatch" rule (preferring more specific type). But the `int(64)` arg prefers `real(64)` due to the "same numeric width" rule.